Core/Graphics/Include/Babylon/Graphics.h

```diff
@@ -18,6 +18,7 @@ namespace Babylon
 
         template<typename NativeWindowT>
         void UpdateWindow(NativeWindowT window);
+        void UpdateWindowType(void* type);
```

Comment:
What is an example of what this type would be?

Reply:
this type looks like its only supposed to be used on linux platforms but it has windows content hidden behind it as well:

https://github.com/bkaradzic/bgfx/blob/e2eb3d56e6c43db6c531194d72e344817e6c4fb0/src/dxgi.cpp#L457

Reply:
this has been updated to provide generics/require custom calls to specify window types